docs/azure/dependency-injection.md

```diff
@@ -109,7 +109,7 @@ In the [Register client](#register-client) section, you explicitly specify the `
 }
 ```
 
-You can add any options from <xref:Azure.Core.ClientOptions> into the JSON file's `AzureDefaults` section. One of the options is the retry policy. For more information, see [Configure a new try policy](#configure-a-new-retry-policy).
```

Comment:
![Yoda's relevant quote](https://gifimage.net/wp-content/uploads/2017/07/do-or-do-not-there-is-no-try-gif-8.gif)